etc/checkstyle.xml

```diff
@@ -165,7 +165,6 @@
 
         <!-- Checks for whitespace                               -->
         <!-- See http://checkstyle.sf.net/config_whitespace.html -->
-        <module name="OperatorWrap"/>
```

Comment:
This is why I was hesitant to add a checkstyle plugin in the first place -- because we waste time arguing about pedantic things like formatting instead of what the code actually is doing. All of the review comments on this PR have been about code style, not what the code is actually doing. 

I am fine with having some generic guidelines about code style like enforcing javadoc and removing unused imports, but I don't want to have every minor code style preference baked into a settings file. 

By removing this check I'm just allowing people to do this thing either way they want, not necessarily my way our your way. In the example above you showed, the operator on a new line makes sense, but lets let the developer do that on a case-by-case basis.

Reply:
for what it's worth, I did make about a dozen changes to my code before opening the PR in order to conform to other checkstyle rules. I think since the checkstyle rules are still pretty new we should be able to tweak the ones that are more minor like this one